modules/addTask.js

```diff
@@ -0,0 +1,23 @@
+import Task from './taskClass.js';
+import TasksUI from './tasksUI.js';
+
+export default class AddTask {
+  // console.log('event works');
+  static addThis() {
+    // console.log('event works');
+    const dB = JSON.parse(localStorage.getItem('tasksDB')) || [];
+    // console.log(dB);
+    const addTaskInput = document.querySelector('#task-input');
+    if (addTaskInput.value) {
+      // console.log('input filled');
+      // console.log(addTaskInput.value);
+      // console.log('database empty');
```

Comment:
- [OPTIONAL] KIndly make sure to remove all comments in keep your codebase clean.